game/static/game/css/preloading.css

```
@@ -0,0 +1,387 @@
+/* ═══════════════════════════════════════════
+   CHECKORA — PRELOADER  |  preloading.css
+   Fonts: Cinzel (title/logo) · Inter (body)
+   ═══════════════════════════════════════════ */
+
+@import url('https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700;900&family=Inter:wght@400;500;600&display=swap');
```

Stylelint (17.12.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700;900&family=Inter:wght@400;500;600&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700;900&family=Inter:wght@400;500;600&display=swap'" (import-notation)

(import-notation)